src/main/scala/viper/silver/verifier/VerificationError.scala

```diff
@@ -153,6 +153,10 @@ trait ErrorMessage {
 
   // Should consider refactoring as a transformer, if/once we make the error message structure recursive
   def withNode(offendingNode: errors.ErrorNode = this.offendingNode) : ErrorMessage
+
+  // Check if the offendingNode contains any `FailureExpectedInfo` info tags
+  def isExpected: Boolean = if (!offendingNode.isInstanceOf[Infoed]) false
```

Comment:
Could be simplified to `offendingNode.isInstanceOf[Infoed] && else-part`  or, more Scala-ish, a pattern-match block.